### PR TITLE
fix output of flux.log

### PR DESCRIPTION
**Pull request by vantelindome**
This pull request is about fix output of flux.log.
It contains below:
- fixed output format of license
- modified print function to debug output
- fixed reference variable of length of nonbonded gpair_table

### CODE REVIEW

**Review by yamato97 — APPROVED** [review]
LGTM. Thank you.

File: curp/compute.py [hunk 1]
```diff
@@ -298,7 +298,7 @@ def init_current(setting, par):
     # Decide Writer.
     t_0 = time.time()
     
-    logger.info("length of gpair_table: {}".format(len(gpair_table)))
+    logger.info("length of gpair_table: {}".format(len(interact_table)))
```
Comment (passive-radio) [inline curp/compute.py:301]: @vantelindome 
`gpair_table` and `interact_table` are not equivalent.
`gpair_table`: group pairs definition
`interact_table`: atom-atom interactions definition

We can output both length of gpair_table and interact_table.

**Review by passive-radio — APPROVED** [review]

**Review by passive-radio — COMMENTED** [review]
@vantelindome 
Please check my comment.

File: curp/compute.py (hunk 1 above)
Comment (vantelindome) [reply curp/compute.py:301]: Thank you for your comment.
The purpose of this info is to show how much the nonbonded atom-atom interaction table is divided, so I have to change the description.

**Comment by vantelindome** [comment]
@passive-radio 
Can you check again? If OK, I'll merge this PR.

File: curp/compute.py (hunk 1 above)
Comment (passive-radio) [reply curp/compute.py]: @vantelindome 

>The purpose of this info is to show how much the nonbonded atom-atom interaction table

In this case, logging text should be
`length of interact_table:` not `length of gpair_table`.

Comment (vantelindome) [reply curp/compute.py]: @passive-radio 
I fixed the text in my latest commit. Please check.

**Review by passive-radio — APPROVED** [review]
LGTM

**Pull request merged**